analysis-pipeline/predictability/fit.conseq

```diff
@@ -0,0 +1,134 @@
+    for model_name, model_config in config.items():
+        output_json = {
+            "model_name": model_name,
+            "screen_name": model_config["Screen"],
+            "data": {}
+        }
+        
+        target = model_config["Target"][0]
+
+        # Choose the correct target based on screen name
+        if model_config["Screen"] == "CRISPR":
+            target_id = "{{ inputs.crispr_gene_effect.source_dataset_id }}"
+        elif model_config["Screen"] == "RNAi":
+            target_id = "{{ inputs.rnai.source_dataset_id }}"
+
+        output_json["data"][target] = {
+            "taiga_id": target_id,
+            "table_type": "target_matrix",
+            "relation": model_config["Relation"]
+        }
+        
+        for feature in model_config["Features"]:
+            if feature.lower() == "lineage":
+                feature_input = {{ inputs.lineage }}
+            elif feature.lower() == "confounder":
+                feature_input = {{ inputs.confounder }}
+            elif feature.lower() == "driverevents":
+                feature_input = {{ inputs.driver_events }}
+            
+            output_json["data"][feature] = {
+                "taiga_id": feature_input["source_dataset_id"],
+                "table_type": feature_input["type"],
+                "dim_type": feature_input["label"].lower(),
+                "required": feature in model_config["Required"],
+                "exempt": False
+            }
+        
+        output_file = f"{model_name.lower()}_config.json"
+        output_file_path = os.path.abspath(output_file)
+
+        with open(output_file, 'w') as f:
+            json.dump(output_json, f, indent=2)
+        print(f"Created {output_file}")
+
+        
+        docker_command = [
+            "docker", "run",
+            "-w", "/daintree",
+            "--pull=always",
+            "-v", f"{output_file_path}:/daintree/{output_file}",
+            "us.gcr.io/broad-achilles/daintree-sparkles:v4",
+            "/install/depmap-py/bin/python3.9", "-u", "run_fit_models.py",
+            "collect-and-fit-generate-config",
+            "--input-files", output_file,
+            "--sparkles-path", "/install/sparkles/bin/sparkles",
+            "--sparkles-config", "/daintree/sparkles-config",
+            "--save-dir", "/daintree/output_data/nayeem-test",
+            "--test", "True",
+            "--skipfit", "False"
+        ]
+
+        try:
+            result = subprocess.run(
+                docker_command,
+                check=True,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                text=True
+            )
+            
+            print("STDOUT:", result.stdout)
+            print("STDERR:", result.stderr)
+
+        except subprocess.CalledProcessError as e:
+            print(f"Command failed with return code {e.returncode}")
+            print("STDOUT:", e.stdout)
+            print("STDERR:", e.stderr)
+    """
```

Comment:
Concretely, if you were to take all my suggestions this would simplify down to:
```
        docker_command = [
            "docker", "run",
            "-v", f"{os.getcwd()}:/work",
            "us.gcr.io/broad-achilles/daintree-sparkles:v4",
            "daintree",
            "collect-and-fit-generate-config",
            "--input-files", f"/work/{output_file}",
            "--sparkles-config", "/work/sparkles-config",
            "--save-dir", "/work/output_data/nayeem-test",
            "--test", "True",
        ]

        subprocess.run(
            docker_command,
            check=True,
        )